CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 - Fetch child identifier in view record
 - Auth token, ip address, remote address redacted from server log
 - **Align Patient data model with FHIR**: Previously we were using `string[]` for `Patient.name.family` field instead of `string` as mentioned in the FHIR standard. We've now aligned the field with the standard.
+- Changed multiple class components into functional components [#993](https://github.com/opencrvs/opencrvs-core/issues/993)
```

Comment:
```suggestion
- Refactored multiple class components into functional ones to facilitate the usage of `useWindowSize` hook [#993](https://github.com/opencrvs/opencrvs-core/issues/993)
```

Reply:
For such a internal change, we can also omit the changelog mention completely. These types of changes do not really affect country implementors or users of OpenCRVS

Reply:
noted, but will it not affect the merging if needed as a requirement